### PR TITLE
Fixed #7 and #8

### LINKED ISSUE #7: Makefile lint command
Suggest to add more linters here

 at least:

- go vet (https://golang.org/cmd/vet/)
- golint (golang.org/x/lint/golint)
- errcheck (https://github.com/kisielk/errcheck)

in advance:
- golangci-lint (https://github.com/golangci/golangci-lint - the best one, cause it aggregates a lot of linters and run them really fast)

### PR DESCRIPTION
**Pull request by ivanterekh**
Implemented suggestions from issues #7 and #8

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,7 +26,7 @@ test:lint
 
 .PHONY: lint
 lint:
-	@go fmt 
-	@goimports -w .
-	@GO111MODULE=on go vet
-	
+	@go fmt ./...
+	@gogroup -order std,other,prefix=github.com/ivanterekh/ -rewrite $(shell find . -type f -name "*.go")
```
Comment (obalunenko) [inline Makefile:30]: suggest to add check if binary exist in other case make will return error

File: Makefile [hunk 2]
```diff
@@ -24,7 +26,7 @@ test:lint
 
 .PHONY: lint
 lint:
-	@go fmt 
-	@goimports -w .
-	@GO111MODULE=on go vet
-	
+	@go fmt ./...
+	@gogroup -order std,other,prefix=github.com/ivanterekh/ -rewrite $(shell find . -type f -name "*.go") 
+	@GO111MODULE=on golangci-lint run
```
Comment (obalunenko) [inline Makefile:31]: the same as above

File: Makefile [hunk 3]
```diff
@@ -24,7 +26,7 @@ test:lint
...
+	@golint ./...
```
Comment (obalunenko) [inline Makefile:32]: the same as above

**Comment by obalunenko** [comment]
Also you can move all linters run to separate bash script and just run it from makefile

File: Makefile (hunk 2 above)
Comment (obalunenko) [inline Makefile:31]: suggest to add config file for golangci-lint where specified what linters should be executed
https://github.com/golangci/golangci-lint#config-file

File: Makefile [hunk 4]
```diff
@@ -24,7 +26,7 @@ test:lint
 
 .PHONY: lint
 lint:
```
Comment (dzyanis) [inline Makefile:28]: I think you can try to use it for pre-hooks https://git-scm.com/book/ru/v1/%D0%9D%D0%B0%D1%81%D1%82%D1%80%D0%BE%D0%B9%D0%BA%D0%B0-Git-%D0%9F%D0%B5%D1%80%D0%B5%D1%85%D0%B2%D0%B0%D1%82%D1%87%D0%B8%D0%BA%D0%B8-%D0%B2-Git

**Review by dzyanis — APPROVED** [review]

**Review by obalunenko — COMMENTED** [review]
Please take a look on my comments

**Pull request merged**